geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExecuteFunctionCommand.java

```diff
@@ -84,8 +85,10 @@ public Result executeFunction(
     }
 
     for (DistributedMember member : dsMembers) {
-      executeAndGetResults(functionId, filterString, resultCollector, arguments, member,
-          resultTable, onRegion);
+      if (!executeAndGetResults(functionId, filterString, resultCollector, arguments, member,
```

Comment:
we probably can use GfshCommand and ResultBuilder's utilities here. This would get rid of a lot of the boiler plate code:

// create the function and build the args first
List<CliFunctionResult> results = executeAndGetFunctionResult(function, args, dsMembers);
return ResultBuilder.buildResult(results);